test/browser/test_glfw3_hi_dpi_aware.c

```diff
@@ -51,6 +51,22 @@ static void checkWindowSize(GLFWwindow *window, int expectedWidth, int expectedH
   assert(fbw == (int) (expectedWidth * ratio) && fbh == (int) (expectedHeight * ratio));
 }
 
+static void checkCanvasSize(int expectedWidth, int expectedHeight)
```

Comment:
Opening braces on the same like as the declaration.

Reply:
Sorry :(

Reply:
fixed